src/Authentication/UnauthorizedServer.php

```diff
@@ -62,6 +62,6 @@ protected function handle_error( WP_Error $error ) {
 			wp_kses_data( $error->get_error_message() ),
 			esc_html__( 'Authentication Required', 'satispress' ),
 			[ 'response' => HTTP::UNAUTHORIZED ]
-		);
+		); // WPCS: XSS OK.
```

Comment:
Use the `// phpcs:ignore ...` style of comments, instead of the WPCS-specific ones.